great_expectations/expectations/core/expect_column_values_to_not_be_null.py

```diff
@@ -78,6 +83,12 @@ class ExpectColumnValuesToNotBeNull(ColumnMapExpectation):
         [expect_column_values_to_be_null](https://greatexpectations.io/expectations/expect_column_values_to_be_null)
     """
 
+    min_value: Union[float, dict, datetime, None] = None
+    max_value: Union[float, dict, datetime, None] = None
+
+    _min_val = validator("min_value", allow_reuse=True)(validate_eval_parameter_dict)
+    _max_val = validator("max_value", allow_reuse=True)(validate_eval_parameter_dict)
```

Comment:
Nowhere in our logic do we check that `min_val >= max_val` - I think we can follow up with root validators here?